### PR TITLE
Yoink

### PR DESCRIPTION
**Pull request by lilyclements**

### CODE REVIEW

File: instat/static/InstatObject/R/Backend_Components/summary_functions.R [hunk 1]
```diff
@@ -417,15 +420,14 @@ instat_object$set("public", "summary_table", function(data_name, columns_to_summ
...
-    #print(as.formula(paste(row_formula, "~", column_formula)))
     cell_values <- dcast(formula = as.formula(paste(row_formula, "~", column_formula)), value.var = "Value", data = cell_values)
     if(include_margins) {
       margin_tables <- list()
       power_sets <- rje::powerSet(factors)
       power_sets <- power_sets[-(c(length(power_sets)))]
       for(facts in power_sets) {
         if(length(facts) == 0) facts <- c()
-        margin_tables[[length(margin_tables) + 1]] <- self$calculate_summary(data_name = data_name, columns_to_summarise = columns_to_summarise, summaries = summaries, factors = facts, store_results = store_results, drop = drop, na.rm = na.rm, return_output = return_output)
+        margin_tables[[length(margin_tables) + 1]] <- self$calculate_summary(data_name = data_name, columns_to_summarise = columns_to_summarise, summaries = summaries, factors = facts, store_results = store_results, drop = drop, na.rm = na.rm, return_output = TRUE)
```
Comment (lilyclements) [inline instat/static/InstatObject/R/Backend_Components/summary_functions.R]: Does this mean we are always running return_output = TRUE so I should remove that option from the dialog?

File: instat/static/InstatObject/R/Backend_Components/summary_functions.R [hunk 2]
```diff
@@ -339,7 +339,7 @@ summary_median <- function(x, na.rm = FALSE,...) {
   return(median(x, na.rm = na.rm))
 }
 
-instat_object$set("public", "summary_table", function(data_name, columns_to_summarise = NULL, summaries, row_factors = c(), column_factors = c(), store_results = TRUE, drop = TRUE, na.rm = FALSE, summary_name = NA, include_margins = FALSE, return_output = FALSE, treat_columns_as_factor = FALSE, page_by = c(), ...) {
+instat_object$set("public", "summary_table", function(data_name, columns_to_summarise = NULL, summaries, row_factors = c(), column_factors = c(), store_results = TRUE, drop = TRUE, na.rm = FALSE, summary_name = NA, include_margins = FALSE, return_output = FALSE, treat_columns_as_factor = FALSE, page_by = c(), as_html = TRUE, signif_fig = 2, ...) {
```
Comment (lilyclements) [inline instat/static/InstatObject/R/Backend_Components/summary_functions.R]: @dannyparsons out of interest, is this "signif_fig" our new "digits" in the dialog?

Comment (lilyclements) [inline instat/static/InstatObject/R/Backend_Components/summary_functions.R]: Well, I've done that now so I hope your answer is yes

Comment (lilyclements) [inline instat/static/InstatObject/R/Backend_Components/summary_functions.R]: Also why do we run return_output = FALSE by default. Can I make this return_output = TRUE by default

Did we also agree for this to not be offered as a checkbox?

Comment (dannyparsons) [inline instat/static/InstatObject/R/Backend_Components/summary_functions.R]: significant figures is what it actually does I think so I changed it.

Yes I only made it FALSE for testing.

**Pull request merged**